Backend/homepage

sayamaeda opened the pull request:
Modified the statement in homepage.
Due to that, added some functions and relationships into Category model and User model, and also added a column in users table.

sayamaeda committed:
[backend/delete-category-modal]

Added function to delete category in Category Controller.

Added use SoftDeletes in Category model and Item model.

Modified onDelete('cascade') in items table.

sayamaeda committed:
[backend/each-category-page]

Added function to show items for each category in Category Controller.

Modified route for each category page.

Added Item Seeder to show each category page.

sayamaeda committed:
[backend/each-category-page]

Added pagination but it doesn't work yet

sayamaeda committed:
[backend/homepage]

Added functions and relationships to User and Category models.

Added a colum for user id to category table.

Modified the statement in homepage.

KobeSergio commented:
@sayamaeda please pull from main to this branch and solve conflicts at app/Models/User.php

thanks!

sayamaeda committed:
[backend/homepage]

Modified the validation in Category controller to make users can add same category names.

sayamaeda committed:
Merge branch 'main' into backend/homepage

sayamaeda commented:
I solved a conflict but I closed pull request because I had something to modify. Sorry
